File: 222/backend/src/main/java/com/campus/secondhand/entity/Review.java
package com.campus.secondhand.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("review")
public class Review {
    
    @TableId(type = IdType.AUTO)
    private Long id;
    
    private Long orderId;
    
    private Long buyerId;
    
    private Long sellerId;
    
    private Integer rating;
    
    private String content;
    
    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createTime;
}

File: 222/frontend/src/api/review.js
import request from './request'

export const addReview = (orderId, sellerId, rating, content) => {
  return request.post(`/review/add/${orderId}`, { sellerId, rating, content })
}

export const getSellerReviews = (sellerId, page = 1, size = 10) => {
  return request.get(`/review/seller/${sellerId}`, { params: { page, size } })
}